chore(docs): clean up and add new content (#15003)
immich-app/immich: docs/docs/features/folder-view.md

# Folder View

> Folder view feature provides an additional view besides the timeline, that is similar file explorer UI to navigate through the folders and files in the library. This feature is handy for a highly curated and customized external library.

Reviewer: ```suggestion
Folder view provides an additional view besides the timeline that is similar to a file explorer. It allows you to navigate through the folders and files in the library. This feature is handy for a highly curated and customized external library or a nicely configured storage template.
```

> You can enable this feature under `Account Settings > Features > Folder View`

Reviewer: ```suggestion
You can enable this feature under [`Account Settings > Features > Folder View`](https://my.immich.app/user-settings?isOpen=feature+folders)
```

## Enable folder view

<img src={require('./img/folder-view-enable.png').default} width="80%" title='Folder view enable' />

## Usage

> You can then navigate the view from the sidebar to explore the folders and files in the library.

Reviewer: ```suggestion
You can then navigate to the view from the sidebar to explore the folders and files in your library.
```
